src/ci/docker/x86_64-gnu-tools/checktools.sh

```diff
@@ -6,7 +6,7 @@ X_PY="$1"
 TOOLSTATE_FILE="$(realpath $2)"
 OS="$3"
 COMMIT="$(git rev-parse HEAD)"
-CHANGED_FILES="$(git diff --name-status HEAD HEAD^)"
+CHANGED_FILES="$(git diff --name-status HEAD HEAD^ || echo)"
```

Comment:
Just to note - I wasn't really sure what was trigger this.

Reply:
This is strange, also if this `git diff` cannot be correctly computed the `checktools.sh` cannot detect tool regression.

Reply:
Hmm my guess is that I added this when I didn't have submodules properly configured, but later got submodules working. Probably best to revert.

Sorry I don't recall exactly why - a lot of this was while I was semi-sleep-deprived on parental leave 😁